client/src/components/Login/Login.jsx

```diff
@@ -4,40 +4,60 @@ import FormControlLabel from '@mui/material/FormControlLabel';
 import Checkbox from '@mui/material/Checkbox';
 import jwt_decode from "jwt-decode";
 import { GoogleLogin, googleLogout } from '@react-oauth/google';
+const axios = require('axios').default;
 
 class Login extends React.Component {
   constructor(props) {
     super(props);
   }
 
-
   responseMessage = (response) => {
     var decoded = jwt_decode(response.credential);
-    console.log('success', response, decoded);
+    //console.log('success', response, decoded);
+    axios.post('/login', {
+      email: decoded.email,
+      credential: response.credential,
+    })
+    .then((response) =>{
+      console.log('success', response);
+      this.setState({user: decoded.email});
+      this.props.updateUser(decoded.email);
+    })
+    .catch((error) => {
+      console.log('fail', error);
```

Comment:
Consider letting the user know their log in failed so they can try again. Use a toast or a paragraph element